Solved Query-33 Issue #20

Krishna-Pradeep-Dawalkar opened the pull request:
Created 'Query-33 Issue #20' folder and in that folder created 'Query-33 Issue #20.md 'file to solve the assigned issue to me i.e. Query-33 Issue #20.

Krishna-Pradeep-Dawalkar committed:
Solved Query-33 Issue #20

Created Query-33 Issue #20 to solve the assigned issue to me i.e. Solved Query-33 Issue #20.

ananya0504 reviewed (changes requested):
There is no Employee table @Krishna-Pradeep-Dawalkar.

Krishna-Pradeep-Dawalkar committed:
Updated file as per the changes expected.

Updated as per the changes expected and replaced the employee table by table(generalized one).
